docker/docker-compose.yml

```diff
@@ -16,10 +17,12 @@ services:
         MM_MIN_REV: 9f867b3
         PHENIX_WEB_AUTH: disabled
     command:
-    - phenix
-    - ui
-    - --hostname-suffixes=-minimega,-phenix
-    - --minimega-console
+      - phenix
+      - ui
+      - --hostname-suffixes=-minimega,-phenix
+      - --logs.minimega-path=/var/log/phenix/minimega.log
+      - --logs.publish-to-ui=info
```

Comment:
I'm not sure I'd suggest having this in the default config. We've seen browser performance issues with pushing logs to the UI. If you really want this in the default config, I'd suggest a higher log level, like `error`.

Reply:
Yeah, although to be fair we saw them with debug level logging so that was likely the main issue... lots of debug logs from phenix and minimega. It would probably be okay to at least have them in there for error or warning logs, that way something shows up in the UI by default.